docs/topics/whatsnew-eap.md

```diff
@@ -199,23 +199,48 @@ kotlin {
 
 Learn more about the [hierarchical project structure in Kotlin Multiplatform](multiplatform-hierarchy.md).
 
-### Deprecated compatibility with Gradle Java plugins
+### Deprecated compatibility with Kotlin Multiplatform Gradle plugin and Gradle Java plugins
 
-Due to compatibility issues between the Kotlin Multiplatform Gradle plugin and the Gradle plugins 
-[Java](https://docs.gradle.org/current/userguide/java_plugin.html), 
-[Java Library](https://docs.gradle.org/current/userguide/java_library_plugin.html), 
-and [Application](https://docs.gradle.org/current/userguide/application_plugin.html), 
-Kotlin %kotlinEapVersion% introduces a deprecation warning when you apply these plugins in the same project.
-In future Kotlin releases, the warning will be raised to an error.
+In Kotlin %kotlinEapVersion%, we introduce a deprecation warning when you apply the Kotlin Multiplatform Gradle plugin 
+and any of the following Gradle Java plugins within the same project: [Java](https://docs.gradle.org/current/userguide/java_plugin.html),
+[Java Library](https://docs.gradle.org/current/userguide/java_library_plugin.html), and [Application](https://docs.gradle.org/current/userguide/application_plugin.html).
+This warning also appears when another Gradle plugin in your multiplatform project applies a Gradle Java plugin. 
```

Comment:
```suggestion
The warning also appears when another Gradle plugin in your multiplatform project applies a Gradle Java plugin. 
```

Because I can.